OpenNefia.Content/Resources/Prototypes/Elona/Portrait.yml

```diff
@@ -6,230 +6,263 @@
 
 - type: Elona.Portrait
   id: Elona.Man1
+  gender: Male
   hspIds:
     elona122: 0
   image:
     filepath: /Graphic/Elona/face1.bmp
     region: 0, 0, 48, 72
 - type: Elona.Portrait
   id: Elona.Man2
+  gender: Male
   hspIds:
     elona122: 1
   image:
     filepath: /Graphic/Elona/face1.bmp
     region: 48, 0, 48, 72
 - type: Elona.Portrait
   id: Elona.Man3
+  gender: Male
   hspIds:
     elona122: 2
   image:
     filepath: /Graphic/Elona/face1.bmp
     region: 96, 0, 48, 72
 - type: Elona.Portrait
   id: Elona.Man4
+  gender: Male
   hspIds:
     elona122: 3
   image:
     filepath: /Graphic/Elona/face1.bmp
     region: 144, 0, 48, 72
 - type: Elona.Portrait
   id: Elona.Man5
+  gender: Male
   hspIds:
     elona122: 4
   image:
     filepath: /Graphic/Elona/face1.bmp
     region: 192, 0, 48, 72
 - type: Elona.Portrait
   id: Elona.Man6
+  gender: Male
   hspIds:
     elona122: 5
   image:
     filepath: /Graphic/Elona/face1.bmp
     region: 240, 0, 48, 72
 - type: Elona.Portrait
   id: Elona.Man7
+  gender: Male
   hspIds:
     elona122: 6
   image:
     filepath: /Graphic/Elona/face1.bmp
     region: 288, 0, 48, 72
 - type: Elona.Portrait
   id: Elona.Man8
+  gender: Male
   hspIds:
     elona122: 7
   image:
     filepath: /Graphic/Elona/face1.bmp
     region: 336, 0, 48, 72
 - type: Elona.Portrait
   id: Elona.Man9
+  gender: Male
   hspIds:
     elona122: 8
   image:
     filepath: /Graphic/Elona/face1.bmp
     region: 384, 0, 48, 72
 - type: Elona.Portrait
   id: Elona.Man10
+  gender: Male
   hspIds:
     elona122: 9
   image:
     filepath: /Graphic/Elona/face1.bmp
     region: 432, 0, 48, 72
 - type: Elona.Portrait
   id: Elona.Man11
+  gender: Male
   hspIds:
     elona122: 10
   image:
     filepath: /Graphic/Elona/face1.bmp
     region: 480, 0, 48, 72
 - type: Elona.Portrait
   id: Elona.Man12
+  gender: Male
```

Comment:
`randomGender`, to make it clear this is used for random generation, and not a general-purpose annotation.

Reply:
changed in 5da0180